Makefile

```
@@ -60,25 +89,17 @@ dist-zip: clean-test clean-dist
 	zip -qr $(DIST).zip $(DIST)
 
 # run tests in the distdir
-dist-test: dist-zip install-vendor-dev
+dist-test: dist-zip install-vendor
 	$(PHPUNIT) --do-not-cache-result --no-logging --configuration=$(DIST)/phpunit.xml
 
 ###
 # install vendor rules
 ###
 install-vendor:
 	if [[ ! -f vendor/autoload.php && ! -f ../../../vendor/autoload.php ]]; then \
-		$(COMPOSER) install $(COMPOSER_INSTALL_FLAGS) --no-dev
-	fi
-
-install-vendor-dev:
-	if [[ ! -f vendor/autoload.php && ! -f ../../../vendor/autoload.php ]]; then \
-		$(COMPOSER) install $(COMPOSER_INSTALL_FLAGS); \
+		$(COMPOSER) install $(COMPOSER_INSTALL_FLAGS) $(VENDOR_FLAGS)
```

This doesn't well equally well with all shells, I'd suggest to be more straightforward:

```suggestion
	test -f vendor/autoload.php \
	    || test -f ../../../vendor/autoload.php \
	    || $(COMPOSER) install $(COMPOSER_INSTALL_FLAGS) $(VENDOR_FLAGS)
```

---

Is there any real need to test for the existence of the files?